web/olga/charts/static/charts/graphs.js

```diff
@@ -38,23 +38,89 @@
      */
     function appendChartData(chart, chart_data, chart_title) {
         layout = {
-          xaxis: {
-            title: chart_title
-          }
+            xaxis: {
+                title: chart_title,
+
+            },
+            yaxis: {
+                nticks: 4,
+                tickfont: {color: '#70A3FF'},
+                showline: true
```

Comment:
Trailing comma is nice to have after the last element in the data container.

Reply:
All done